recipe/meta.yaml

```diff
@@ -52,8 +52,12 @@ about:
   home: https://pypi.org/project/unearth/
   summary: A utility to fetch and download python packages
   license: MIT
-  license_file:
-    - LICENSE
+  license_family: MIT
+  license_file: LICENSE
+  license_url: https://github.com/frostming/unearth/blob/main/LICENSE
+  dev_url: https://github.com/frostming/unearth
+  doc_url: https://unearth.readthedocs.io/en/latest/
+  doc_source_url: https://github.com/frostming/unearth/tree/main/docs
```

Comment:
`doc_url` and `doc_source_url` are redundant. Same with `license_file` and `license_url`. These messages will be removed in the next linter release.